app/secrets.yml

```diff
@@ -1,3 +1,9 @@
-DB_USERNAME: !var pcf/app/database/username
-DB_PASSWORD: !var pcf/app/database/password
+ORG_DB_USERNAME: !var pcf/org/database/username
```

Comment:
This file is being copied over from the v4 / v5 versions during the script run, right? We should `.gitignore` and delete it.